### ISSUE TITLE
[ERR_INVALID_ARG_TYPE]: The "path" argument must be of type string. Received undefined

### ISSUE THREAD

**Issue by danielb1986** [naddison36/sol2uml#83]
I tried many different ways to call this package but I just cant get it to work and I dont know why. Just installed it and followed the instructions but I keep getting this error, regardless of what I pass on as argument. 

Any clue what I am doing wrong?

<img width="824" alt="image" src="https://user-images.githubusercontent.com/77058885/181877965-4935cf52-7557-4bdf-8c04-c7d0cb2fca98.png">

**Comment by 0xged** [comment]
Ditto.

**Comment by engn33r** [comment]
Same here. I do get an svg generated when I use an address from mainnet so the error appears only with local paths.

**Comment by naddison36** [comment]
Thanks for reporting this. I'll look at it now.

**Comment by naddison36** [comment]
ok, I can replicate. The workaround is to specify an output file with the `-o, --outputFileName <value>` option. For example

```
sol2uml class ./contract -o contract.svg
```

I'll fix setting a default output filename in the next release.

**Comment by naddison36** [comment]
Alternatively, you can specify the base contract name with the `-b, --baseContractNames <value>` option. This will draw all the dependencies from the specified base contract and output to the same filename. For example, the following will output to `MyContract.svg`.

```
sol2uml class ./contract -b MyContract
```

**Comment by naddison36** [comment]
this should be fixed in v2.1.1. Please reopen this issue if it's not working for you.

Thanks
